travvy/src/components/AttractionsList.vue

```diff
@@ -85,35 +85,83 @@
       <h1>Map</h1> 
       <div id="map">
         <div v-if="$store.state.city == 'Toronto'">
-        <gmap-map v-bind:center="{lat: 43.6532, lng: -79.3832}" v-bind:zoom="11" style="height: 425px">
+        <gmap-map v-bind:center="{lat: 43.6532, lng: -79.3832}"
+          v-bind:zoom="11"
+          style="margin:45px; height: 425px;">
+          <GmapMarker
+            v-for="value in $store.state.recommendedAttractions" 
+            :key="value.id"
+            :position= "{lat:value.lat,lng:value.long}"
```

Comment:
Great you were able to figure out how to access the lat/long values from the store!

Reply:
yep! Daria helped me out.